Translations/ShellExtension/Turkish.po

```diff
@@ -1,15 +1,18 @@
-# This file is part from WinMerge <https://winmerge.org/>
+# This file is part from WinMerge <http://winmerge.org/>
```

Comment:
Is this fix intended or an accidental fix?

Reply:
Sorry accidental. I just missed. Should be https. Thanks.

Reply:
Thank you for your reply.
I'll fix that later.